cfn-resources/maintenance-window/README.md

```diff
@@ -0,0 +1,60 @@
+# Mongodb::Atlas::MaintenanceWindow
+
+## Description
+The MaintenanceWindow resource provides access to retrieve or update the current Atlas project maintenance window.
+
+## Attributes & Parameters
+
+Please consult the [Resource Docs](docs/README.md)
+
+## Local Testing
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+repo_root=$(git rev-parse --show-toplevel)
+source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py)
+cd ${repo_root}/cfn-resources/project
+./test/project.create-sample-cfn-request.sh YourProjectName > test.request.json 
+echo "Sample request:"
+cat test.request.json
+cfn invoke CREATE test.request.json 
+cfn invoke DELETE test.request.json 
+```
+
+Both CREATE & DELETE tests must pass.
+
+## Installation
+TAGS=logging make
+cfn submit --verbose --set-default
+
+## Usage
+
+The [/quickstart-mongodb-atlas/scripts/launch-x-quickstart.sh](launch-x-quickstart.sh) script
```

Comment:
This link doesn't work, can you fix?

Reply:
yes Zuhair! i see this file is nowhere in the project, and is mentioned in all the readmes, ill take a look at it, for now ill just remove this part

Reply:
I realized that it is wrong in all the readme files, i fixed the others too, thanks!